ccmlib/cmds/cluster_cmds.py

```diff
@@ -541,6 +544,8 @@ def get_parser(self):
 
     def validate(self, parser, options, args):
         Cmd.validate(self, parser, options, args, load_cluster=True)
+        if self.options.no_wait and self.wait_for_binary_proto:
+            print_("WARN: Both --wait-for-binary-proto and --no-wait were specified. --no-wait will be preferred.")
```

Comment:
That is what happened. Except I had to leave the --wait-other-notice for backwards compatibility.